tests/Silex/Tests/Provider/FormServiceProviderTest.php

```diff
@@ -53,6 +53,51 @@ public function testFormServiceProviderWillLoadTypes()
         $this->assertInstanceOf('Symfony\Component\Form\Form', $form);
     }
 
+    public function testFormServiceProviderWillLoadTypesServices()
+    {
+        $app = new Application();
+
+        $app->register(new FormServiceProvider());
+
+        $app['dummy'] = function () {
+            return new DummyFormType();
+        };
+        $app->extend('form.types', function ($extensions) {
+            $extensions[] = 'dummy';
+
+            return $extensions;
+        });
+
+        $form = $app['form.factory']
+            ->createBuilder(class_exists('Symfony\Component\Form\Extension\Core\Type\RangeType') ? 'Symfony\Component\Form\Extension\Core\Type\FormType' : 'form', array())
```

Comment:
RangeType always exists as we bumped the min requirement to 2.8 (same everywhere else).